docs/Deploy/Reference-Architectures.md

```diff
@@ -2,13 +2,13 @@
 
 ## The Fleet binary
 
-The Fleet application contains two single static binaries which provide web based administration, REST API, and CLI interface to Fleet.
+The Fleet application contains two single static binaries which provide web based administration, REST API, and [CLI interface](https://fleetdm.com/docs/using-fleet/fleetctl-cli) to Fleet.
 
 The `fleet` binary contains:
-- The Fleet TLS web server (no external webserver is required but it supports a proxy if desired)
+- The [Fleet TLS web server](https://fleetdm.com/docs/configuration/fleet-server-configuration) (no external webserver is required but it supports a proxy if desired)
 - The Fleet web interface
 - The Fleet application management [REST API](https://fleetdm.com/docs/using-fleet/rest-api)
-- The Fleet osquery API endpoints
+- The [Fleet osquery API endpoints](https://fleetdm.com/docs/rest-api/rest-api) which enables FleetDM's functionality powered by osquery
```

Comment:
Can you remove this change? The bullet point above it is already has a link to the REST API documentation.
```suggestion
- The Fleet osquery API endpoints
```

Reply:
Well spotted @eashaw thank you. I would've rather kept the text after the link but I hope the update as requested works